src/lambdas/getProductById.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+const dbHelper = require("../helpers/db.helper");
+const { responseHeaders } = require("../common/config");
+const { StatusCodes } = require("http-status-codes");
+const validate = require("uuid-validate");
+
+const UUID_VERSION = 4;
+
+const getProductByIdQuery = (id) => `
+  select products.id, products.title, products.description, products.price, products.image, stocks."count" from products 
+  left join stocks
+  on stocks.product_id = products.id
+  where products.id = '${id}'
+`;
+
+const getProductById = async (event) => {
+  let dbConnection = null;
+
+  try {
+    let id = null;
+    if (event.pathParameters) {
+      id = event.pathParameters.id;
+    }
+    if (!id)
+      return {
+        headers: responseHeaders,
+        statusCode: StatusCodes.FORBIDDEN,
```

Comment:
I think we should use the HTTP status code `400`, which is `Bad request`. `Forbidden` means that either you unauthorized or unauthenticated in the system.